src/WireMock.Net.Aspire/WireMockServerBuilderExtensions.cs

```diff
@@ -1,175 +1,222 @@
+    {
+        Guard.NotNull(wiremock);
+
+        wiremock.Resource.Arguments.AdminUsername = Guard.NotNull(username);
+        wiremock.Resource.Arguments.AdminPassword = Guard.NotNull(password);
+        return wiremock;
+    }
+
+    /// <summary>
+    /// Use WireMock Client's AdminApiMappingBuilder to configure the WireMock.Net resource.
+    /// </summary>
+    /// <param name="wiremock">The <see cref="IResourceBuilder{WireMockServerResource}"/>.</param>
+    /// <param name="configure">Delegate that will be invoked to configure the WireMock.Net resource.</param>
+    /// <returns></returns>
+    public static IResourceBuilder<WireMockServerResource> WithApiMappingBuilder(this IResourceBuilder<WireMockServerResource> wiremock, Func<AdminApiMappingBuilder, Task> configure)
+    {
+        return wiremock.WithApiMappingBuilder((adminApiMappingBuilder, _) => configure.Invoke(adminApiMappingBuilder));
+    }
+
+    /// <summary>
+    /// Use WireMock Client's AdminApiMappingBuilder to configure the WireMock.Net resource.
+    /// </summary>
+    /// <param name="wiremock">The <see cref="IResourceBuilder{WireMockServerResource}"/>.</param>
+    /// <param name="configure">Delegate that will be invoked to configure the WireMock.Net resource.</param>
+    /// <returns></returns>
+    public static IResourceBuilder<WireMockServerResource> WithApiMappingBuilder(this IResourceBuilder<WireMockServerResource> wiremock, Func<AdminApiMappingBuilder, CancellationToken, Task> configure)
+    {
+        Guard.NotNull(wiremock);
+
+        wiremock.ApplicationBuilder.Services.TryAddLifecycleHook<WireMockServerLifecycleHook>();
+        wiremock.Resource.Arguments.ApiMappingBuilder = configure;
+
+        return wiremock;
+    }
+
+    /// <summary>
+    /// Enables the WireMockInspect, a cross-platform UI app that facilitates WireMock troubleshooting.
+    /// This requires installation of the WireMockInspector tool.
+    /// <code>
+    /// dotnet tool install WireMockInspector --global --no-cache --ignore-failed-sources
+    /// </code>
+    /// </summary>
+    /// <param name="builder">The <see cref="IResourceBuilder{WireMockNetResource}"/>.</param>
+    /// <returns></returns>
+    public static IResourceBuilder<WireMockServerResource> WithOpenInspectorCommand(this IResourceBuilder<WireMockServerResource> builder)
+    {
+        Guard.NotNull(builder);
+
+        CommandOptions commandOptions = new()
+        {
+            Description = "Requires installation of the WireMockInspector tool:\ndotnet tool install WireMockInspector --global --no-cache --ignore-failed-sources",
+            UpdateState = OnUpdateResourceState,
+            IconName = "BoxSearch",
+            IconVariant = IconVariant.Filled
+        };
+
+        builder.WithCommand(
+            name: "open-inspector",

```

Comment:
probably "open-inspector" should be like "wiremock-inspector" ???
and
"Open Inspector" --> "WireMock Inspector"